pkg/drivers/none/none.go

```diff
@@ -220,19 +222,19 @@ func (d *Driver) RunSSHCommandFromDriver() error {
 }
 
 // stopKubelet idempotently stops the kubelet
-func stopKubelet(exec command.Runner) error {
+func stopKubelet(cr command.Runner) error {
 	glog.Infof("stopping kubelet.service ...")
 	stop := func() error {
-		cmdStop := "sudo systemctl stop kubelet.service"
-		cmdCheck := "sudo systemctl show -p SubState kubelet"
-		err := exec.Run(cmdStop)
-		if err != nil {
-			glog.Errorf("temporary error for %q : %v", cmdStop, err)
+		cmdStop := exec.Command("/bin/bash", "-c", "sudo systemctl stop kubelet.service")
```

Comment:
Remove /bin/bash here and elsewhere - it isn't required unless there is shell logic to execute (`|, && or ||`) or variable substitution (`$`)

Reply:
exec.Command expects (name and args) would the name be "sudo" ?